src/auth/oauth2client.ts

```diff
@@ -460,64 +529,40 @@ export class OAuth2Client extends AuthClient {
     this._clientId = opts.clientId;
     this._clientSecret = opts.clientSecret;
     this.redirectUri = opts.redirectUri;
-  }
 
-  protected static readonly GOOGLE_TOKEN_INFO_URL =
-    'https://oauth2.googleapis.com/tokeninfo';
-
-  /**
-   * The base URL for auth endpoints.
-   */
-  private static readonly GOOGLE_OAUTH2_AUTH_BASE_URL_ =
-    'https://accounts.google.com/o/oauth2/v2/auth';
-
-  /**
-   * The base endpoint for token retrieval.
-   */
-  private static readonly GOOGLE_OAUTH2_TOKEN_URL_ =
-    'https://oauth2.googleapis.com/token';
-
-  /**
-   * The base endpoint to revoke tokens.
-   */
-  private static readonly GOOGLE_OAUTH2_REVOKE_URL_ =
-    'https://oauth2.googleapis.com/revoke';
-
-  /**
-   * Google Sign on certificates in PEM format.
-   */
-  private static readonly GOOGLE_OAUTH2_FEDERATED_SIGNON_PEM_CERTS_URL_ =
-    'https://www.googleapis.com/oauth2/v1/certs';
+    this.endpoints = {
+      tokenInfoUrl: `https://oauth2.${this.universeDomain}/tokeninfo`,
```

Comment:
I am not sure if these are in scope?

Reply:
Opened a PR to revert: https://github.com/googleapis/google-auth-library-nodejs/pull/1744